src/services/user/firstLogin.js

```diff
@@ -12,6 +13,19 @@ const getAutomaticConsent = () => ({
 	dateOfTermsOfUseConsent: Date.now(),
 });
 
+const isStudent = (userRoles = []) => userRoles.some((role) => role.name === 'student');
+
+// For admins the birthdate is not set and for teacher is optionl.
+const validateExistingStudentBirthdate = (user, userRoles, data = {}) => {
+	if (!isStudent(userRoles)) {
+		return;
+	}
+
+	if ((user.birthday || data.studentBirthdate) === false) {
+		throw new Error('Bitte ein Geburtstag angeben.');
```

Comment:
"Bitte gib dein Geburtsdatum an" maybe sounds nicer

Reply:
My text can interpret as type some birthday from a person do you know into it. :P 
Yes it is nicer. :D

Reply:
done